### ISSUE TITLE
Boundary event associations can be created but can't be loaded

### ISSUE THREAD

**Issue by AndyBan** [bpmn-io/bpmn-js#1170]
__Describe the Bug__

Associations on `bpmn:BoundaryEvents` can be modeled, bu won't be rendered on import.

Created:
![image](https://user-images.githubusercontent.com/25322348/62955253-a207e700-bde8-11e9-80b5-d49be9b1c133.png)

After load:

![image](https://user-images.githubusercontent.com/25322348/62955487-12166d00-bde9-11e9-8763-56bacfd0f288.png)

__Steps to Reproduce__

Using https://demo.bpmn.io/new
1. Create Task
2. Add signal boundary event (Happens for other boundary events too)
3. Associate boundary event with data object
4. Export to file, refresh, import from file

__Expected Behavior__

Associations on Boundary Events will be rendered correctly.

__Environment__

 - Browser: all
 - OS: macOS
 - Library version: 3, 4, 5

**Comment by pinussilvestrus** [comment]
Was able to reproduce it on Chrome (for several kinds of Boundary Events). It also appears in older `bpmn-js` versions.

**Comment by pinussilvestrus** [comment]
changed the issue description to fit our template.

**Comment by nikku** [comment]
Closed via https://github.com/bpmn-io/bpmn-js/pull/1172.

**Comment by AndyBan** [comment]
@nikku

I have tested using npm@5.0.1 and on https://demo.bpmn.io/ and fix doesn't appear to have worked with edge@77 and firefox@68.

Functionality looks to be the same as before where data association isn't displayed.  The produced xml looks fine though:

```
<?xml version="1.0" encoding="UTF-8"?>
<bpmn:definitions xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:bpmn="http://www.omg.org/spec/BPMN/20100524/MODEL" xmlns:bpmndi="http://www.omg.org/spec/BPMN/20100524/DI" xmlns:dc="http://www.omg.org/spec/DD/20100524/DC" xmlns:di="http://www.omg.org/spec/DD/20100524/DI" id="Definitions_1dldzfo" targetNamespace="http://bpmn.io/schema/bpmn" exporter="bpmn-js (https://demo.bpmn.io)" exporterVersion="5.0.0">
  <bpmn:process id="Process_0akyfq7" isExecutable="false">
    <bpmn:startEvent id="StartEvent_1k7b65z">
      <bpmn:outgoing>SequenceFlow_0ljsapo</bpmn:outgoing>
    </bpmn:startEvent>
    <bpmn:task id="Task_087uhhc">
      <bpmn:incoming>SequenceFlow_0ljsapo</bpmn:incoming>
    </bpmn:task>
    <bpmn:sequenceFlow id="SequenceFlow_0ljsapo" sourceRef="StartEvent_1k7b65z" targetRef="Task_087uhhc" />
    <bpmn:boundaryEvent id="BoundaryEvent_1fbn91z" attachedToRef="Task_087uhhc">
      <bpmn:outgoing>SequenceFlow_0lo5gt4</bpmn:outgoing>
      <bpmn:dataOutputAssociation id="DataOutputAssociation_027acd9">
        <bpmn:targetRef>DataObjectReference_1k5ejq4</bpmn:targetRef>
      </bpmn:dataOutputAssociation>
    </bpmn:boundaryEvent>
    <bpmn:task id="Task_1dryxgw">
      <bpmn:incoming>SequenceFlow_0lo5gt4</bpmn:incoming>
    </bpmn:task>
    <bpmn:sequenceFlow id="SequenceFlow_0lo5gt4" sourceRef="BoundaryEvent_1fbn91z" targetRef="Task_1dryxgw" />
    <bpmn:dataObjectReference id="DataObjectReference_1k5ejq4" dataObjectRef="DataObject_0op3ch8" />
    <bpmn:dataObject id="DataObject_0op3ch8" />
  </bpmn:process>
  <bpmndi:BPMNDiagram id="BPMNDiagram_1">
    <bpmndi:BPMNPlane id="BPMNPlane_1" bpmnElement="Process_0akyfq7">
      <bpmndi:BPMNShape id="_BPMNShape_StartEvent_2" bpmnElement="StartEvent_1k7b65z">
        <dc:Bounds x="156" y="81" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Task_087uhhc_di" bpmnElement="Task_087uhhc">
        <dc:Bounds x="250" y="59" width="100" height="80" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNEdge id="SequenceFlow_0ljsapo_di" bpmnElement="SequenceFlow_0ljsapo">
        <di:waypoint x="192" y="99" />
        <di:waypoint x="250" y="99" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNShape id="BoundaryEvent_1fbn91z_di" bpmnElement="BoundaryEvent_1fbn91z">
        <dc:Bounds x="272" y="121" width="36" height="36" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNShape id="Task_1dryxgw_di" bpmnElement="Task_1dryxgw">
        <dc:Bounds x="360" y="180" width="100" height="80" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNEdge id="SequenceFlow_0lo5gt4_di" bpmnElement="SequenceFlow_0lo5gt4">
        <di:waypoint x="290" y="157" />
        <di:waypoint x="290" y="220" />
        <di:waypoint x="360" y="220" />
      </bpmndi:BPMNEdge>
      <bpmndi:BPMNShape id="DataObjectReference_1k5ejq4_di" bpmnElement="DataObjectReference_1k5ejq4">
        <dc:Bounds x="162" y="245" width="36" height="50" />
      </bpmndi:BPMNShape>
      <bpmndi:BPMNEdge id="DataOutputAssociation_027acd9_di" bpmnElement="DataOutputAssociation_027acd9">
        <di:waypoint x="279" y="153" />
        <di:waypoint x="198" y="251" />
      </bpmndi:BPMNEdge>
    </bpmndi:BPMNPlane>
  </bpmndi:BPMNDiagram>
</bpmn:definitions>
```

**Comment by pinussilvestrus** [comment]
It works perfectly fine to me on yesterday's released version (tag [`v5.0.1`](https://github.com/bpmn-io/bpmn-js/commit/4cda939b4dda0624507ddcd7b865d72a8fccf746)). However, it's not shipped to https://demo.bpmn.io right now.

**Comment by AndyBan** [comment]
Oh I do apologize, had a bit of a caching issue and it is indeed working, thanks  :)
